testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/micrometer/MicrometerRemovalTestCase.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.micrometer;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.arquillian.api.ContainerResource;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.test.shared.TimeoutUtil;
+import org.jboss.dmr.ModelNode;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.plugin.tools.OperationExecutionException;
+import org.wildfly.plugin.tools.server.ServerManager;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class MicrometerRemovalTestCase {
+    private static final ModelNode micrometerExtension = Operations.createAddress("extension", "org.wildfly.extension.micrometer");
+    private static final ModelNode micrometerSubsystem = Operations.createAddress("subsystem", "micrometer");
+    private static final ModelNode ADDRESS_SERVER_LOG_DIR = Operations.createAddress("path", "jboss.server.log.dir");
+    private static final String ERROR_MESSAGE = "Failed to publish metrics to OTLP receiver";
+
+    @ContainerResource
+    private ServerManager serverManager;
+    private String logFilePath;
+
+    @Before
+    public void getLogLocation() throws IOException {
+        ModelNode op = Operations.createReadAttributeOperation(ADDRESS_SERVER_LOG_DIR, "path");
+        logFilePath = Operations.readResult(serverManager.executeOperation(op)).asString();
```

Comment:
We discussed and I know a change is coming, but for others. We know this will fail :)